src/libraries/System.Net.Http.WinHttpHandler/src/System/Net/Http/NetEventSource.WinHttpHandler.cs

```diff
@@ -5,6 +5,11 @@
 
 namespace System.Net
 {
-    [EventSource(Name = "Private.InternalDiagnostics.System.Net.Http.WinHttpHandler")]
-    internal sealed partial class NetEventSource { }
+    [EventSource(Name = NetEventSourceName)]
```

Comment:
I believe you should be able to drop the attributes entirely

Reply:
I've meant that the attribute is not needed anymore in all the places where you're calling the base ctor now.
The WinHttpHandler one should still provide the name in its ctor.

Reply:
Don't mind keeping it around, it's just a few extra bytes.
The static `GetName`/`GetGuid` methods might be the only one that care about the attribute itself, but these are internal EventSource types anyway.